src/STCard.ts

```diff
@@ -0,0 +1,52 @@
+import Card from './models/Card/Card';
```

Comment:
This is throwing some tslint warnings which I think would be best fixing now before they get too large and then setting the errors up to emit errors

Reply:
looks like it shouldn't appear anymore - imports were not alphabetized